### ISSUE TITLE
Feature Request: Option to keep 'gift' cards in deck tracker

### ISSUE THREAD

**Issue by mSterian** [HearthSim/Hearthstone-Deck-Tracker#3918]
Cards that didn't start in my deck are tracked with a gift icon for my opponent's overlay.
It does the same for my deck overlay except as soon as I don't have the card in my hand anymore, it disappears from the tracker.
Please give option to remember the gift cards that I have played.
This is very important for Tess decks.

**Comment by ShadowFox950** [comment]
Yes please me and my girlfriend both use Mage Decks and having a way to know what skills will be cast when using Grand Magister is so important.
